docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-0-2,Logstash 5.1.1>>
+* <<logstash-5-0-1,Logstash 5.1.0>>
+
+[[logstash-5-1-1]]
+=== Logstash 5.1.1 Release Notes
+
+* Persistent Queues is a new beta feature that provides infrastructure in Logstash for buffering event data to disk. 
+  By default, Logstash still uses in-memory, bounded queue between the input and filter stages without any guarantees 
+  for data delivery. Users can now opt-in to enable disk-based persistence that can prevent event loss in certain 
+  scenarios. The setting `queue.type: persisted` enables this feature ({lsissue}2605[Issue 2605], {lsissue}2606[Issue 2606]).
+* Filters can be configured to log event data and related context when it exceeds a specified execution time. 
+  These slowlogs will be collected in a separate file called `logstash-slowlog-plain-YYYY-MM-dd.log` ({lsissue}5733[Issue 5733]).
+* Added two new fields 'id', and 'name' to the base metadata for API requests. The id field is the persisted 
```

Comment:
A few changes to punctuation and tagging. Change to, "Added two new fields, `id` and `name`, to the base metadata for API requests. The `id` field is the persisted UUID. The `name` field is the custom name the user has passed in (defaults to the hostname).